bin/zbrewmount

```diff
@@ -0,0 +1,185 @@
+    must specify either -s or -t, not both
+    -v is optional
+    sw is mandatory
+    
+Examples:
+    Mount IGY630 at ZBREW_SRC_ZFSROOT
+        zbrewmount -m -s IGY630
+    Unmount EQAE20 at ZBREW_TGT_ZFSROOT as well as ZBREW_TGT_HLQ
+        zbrewmount -u -t EQAE20
+" >&2
+    exit 8
+}
+
+# ---------------------------------------------------------------------
+# some options cannot be appeared in multiple times or combining with others
+# ---------------------------------------------------------------------
+function opt_combine_repeat_chk
+{
+    if [ "$1" = "-mu" ]; then
+        if [ ! -z $unmount ] || [ ! -z $mount ]; then
+            echo "**ERROR: cannot have multiple -m and -u or combining them" >&2
+            syntax
+        fi
+    elif [ "$1" = "-st" ]; then
+        if [ ! -z $srcmode ] || [ ! -z $tgtmode ]; then
+            echo "**ERROR: cannot have multiple -s and -t or combining them" >&2
+            syntax
+        fi
+    fi
+}
+
+
+# ---------------------------------------------------------------------
+# --- read startup arguments
+# ---------------------------------------------------------------------
+function read_args
+{
+unset mount unmount srcmode tgtmode verbose sw
+while getopts ":mustv" opt; do
+    case ${opt} in
+        m )
+            opt_combine_repeat_chk -mu
+            mount="true"
+            ;;
+        u )
+            opt_combine_repeat_chk -mu
+            unmount="true"
+            ;;
+        s )
+            opt_combine_repeat_chk -st
+            srcmode="true"
+            ;;
+        t )
+            opt_combine_repeat_chk -st
+            tgtmode="true"
+            ;;
+        v )
+            verbose="true"
+            ;;
+        \?)
+            if [ ${OPTARG} != "?" ]; then
+                echo "Invalid option specified: -""${OPTARG}"
+            fi
+            syntax
+            ;;
+     esac
+done
+shift $(expr $OPTIND - 1 )
+
+if [ -z "$mount" ] && [ -z "$unmount" ]; then
+    echo "** ERROR: must specify mount -m or unmount -u option" >&2
+    syntax
+fi
+
+if [ -z "$srcmode" ] && [ -z "$tgtmode" ]; then
+    echo "** ERROR: must specify -s or -t option" >&2
+    syntax
+fi
+
+sw=$1
+if [ -z $sw ]; then
+    echo "**ERROR: sw name must be specified" >&2
+    syntax
+fi
+shift
+ussname=$(echo ${sw} | tr '[:upper:]' '[:lower:]');
+zosname=$(echo ${sw} | tr '[:lower:]' '[:upper:]');
+prefix=`echo "${ussname}" | awk '{ print substr($1, 0, 3) }'`
+bom_file="${ZBREW_REPOROOT}/zbrew-${prefix}/${ussname}/${ussname}bom.json"
+}
+
+function run_unmount {
+    mountpoints=$1
+    zbrew_zfsroot=$2
+    zbrew_hlq=$3
+    zosname=$4
+
+    for mount in $(echo $mountpoints | sed "s/,/ /g"); do
+        absmount="${zbrew_zfsroot}${mount#*:}"
+        if [ ! -z $verbose ]; then
+            echo "**INFO: Attempting to unmount $absmount"
+        fi
+        if [ -d "${absmount}" ]; then
+            /usr/sbin/unmount -R "${absmount}" 2>/dev/null
```

Comment:
should put the output into a temporary file and, if the rc is non-zero from unmount, print out the details so someone can see why it failed (along with failing the function)